src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -48,6 +49,7 @@
   private TsoRequest tsoReq;
   private volatile LeaderWrapper leaderWrapper;
   private ScheduledExecutorService service;
+  private IsolationLevel isolationLevel;
```

Comment:
It is not a origin Region. It is actual an instance of TiRegion which is a wrapper class we defined in our project. `IsolationLevel` is used in `copIterator` [here] (https://github.com/pingcap/tidb/blob/master/store/tikv/coprocessor.go#L453) when client want to issue a coprocessor call. If we dig more, `IsolationLevel` is actually used in context which you can find [here](https://github.com/pingcap/tidb/blob/master/store/tikv/region_request.go#L80)

Based on these, I add `IsolationLevel` in `PDClient` and `TiRegion`. We set `IsolationLevek` to context in `TiRegion`.